bindata/network/openshift-sdn/sdn.yaml

```diff
@@ -159,11 +159,6 @@ spec:
           preStop:
             exec:
               command: ["rm","-f","/etc/cni/net.d/80-openshift-network.conf", "/host/opt/cni/bin/openshift-sdn"]
-        # this comes from the kube-proxy code
```

Comment:
@squeed do you think this should be removed altogether or we should just tweak around the probe parameters at scale to account for slower response times?

Reply:
The problem (IIRC) is that the kube-proxy liveness server doesn't get started until after the sdn "node" stuff is fully started up, so in large clusters where startup takes longer, the kubelet may time out on the liveness check before it's even running. So the fix, if we want to keep the liveness check, is to rearrange sdn startup. But I think there was some feeling that the liveness check isn't really super helpful anyway...